modules/installation-gcp-organization-policies.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-gcp-organization-policies_{context}"]
+= Configuring {gcp-short} organization policies and VPC service controls
+
+[role="_abstract"]
+{product-title} requires that you modify or remove some organization policies and VPC service controls when installing a cluster on {gcp-first}. If your organization uses the following policies, they must be modified or removed:
+
+* `compute.trustedImageProjects`
+* `iam.allowedPolicyMemberDomains`
+* `storage.publicAccessPrevention`
+* VPC Service Controls egress rules
+
+.Prerequisites
+
+* You created a project to host your cluster.
+
+.Procedure
+
+. Modify the `constraints.compute.trustedImageProjects` constraint in your {gcp-short} project and add `projects/rhcos-cloud` to the `allowedValues` list, as in the following example. Alternatively, you can delete the `constraints.compute.trustedImageProjects` constraint.
++
+[source,yaml]
+----
+constraint: constraints/compute.trustedImageProjects
+listPolicy:
+ allowedValues:
+    - projects/rhcos-cloud
+----
+. Modify the `iam.allowedPolicyMemberDomains` constraint to allow the service account that the installation program uses to authenticate with {gcp-short} and create storage.
+. Disable the `storage.publicAccessPrevention` constraint so that the installation program can access your project's cloud storage.
+. Create an egress rule that allows access to the {op-system} project, as in the following example:
++
+[source,yaml]
+----
+- egressTo:
+    roles:
+    - <role_name>
+    resources:
+    - projects/902460926346
+  egressFrom:
+    identityType: <service_account>>
+----
+where:
++
+<role_name>:: Specifies the IAM role that you created for the installation program.
+<service_account>:: Specifies the name of the installation program service account.
```

Comment:
@lnguyen1401 Do you know which role or permissions are required? 

@bscott-rh Per Google [doc](https://docs.cloud.google.com/vpc-service-controls/docs/ingress-egress-rules#egress-rules-reference), "egressFrom.identityType" should be ANY_IDENTITY, ANY_USER_ACCOUNT, or ANY_SERVICE_ACCOUNT instead. And "egressFrom.identities" can be used to list the service accounts and etc.